Kitodo/src/main/java/org/kitodo/production/interfaces/activemq/MapMessageObjectReader.java

```diff
@@ -11,19 +11,23 @@
 
 package org.kitodo.production.interfaces.activemq;
 
+import java.util.Arrays;
 import java.util.Collection;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.HashSet;
+import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 import javax.jms.JMSException;
 import javax.jms.MapMessage;
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.util.Strings;
```

Comment:
```suggestion
import org.apache.commons.lang3.StringUtils;
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;
```
Import `StringUtils` instead of `Strings` since it provides an explicit numeric check. (see below)